ext/iperf3/cjson.c

```diff
@@ -0,0 +1,3085 @@
+/*
```

Comment:
We already have cjson available in NCS, I wonder if we could use it instead of introducing another copy. Not sure though if this is variant was somehow modified for the iperf usage or not.

Reply:
Let's try if it builds and see if it works. If not, let's use iperf3 version.